src/app/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Entry from '../blog/Entry';
+import mockEntry from '../mockdata';
```

Comment:
Is there a reason, why you changed 'mockdata' to 'mockEntry'? I would recommend same name